src/beamlime/handlers/config_handler.py

```diff
@@ -31,10 +32,28 @@ def message_key(instrument: str) -> MessageKey:
     def __init__(self, *, logger: logging.Logger | None = None, config: Config):
         super().__init__(logger=logger, config=config)
         self._store = config
+        self._actions: dict[str, list[callable]] = {}
 
     def get(self, key: str, default=None):
         return self._store.get(key, default)
 
+    def register_action(
+        self,
+        key: str,
+        callback: callable,
```

Comment:
`callback` sounds like something to be done after the actual action though.
How about just `action`...?

And I think `callable` is a function, not a type. (at least my linter complains)

Reply:
Yes, and I should specify the signature. Ok if I address this in the follow-up #354, where the signature changes?